firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -5324,8 +5324,38 @@ extension String {
     public static let SendCrashReportsSettingMessage = MZLocalizedString(
         key: "", // Settings.CrashReports.Message.v135
         tableName: "Settings",
-        value: "Crash reports allow us to diagnose and fix issues with the browser.",
+        value: "Automatically send crash reports to Mozilla to diagnose and fix issues with the browser. Reports may include personal or sensitive data",
```

Comment:
With past strings we used a wildcard %@ for the app name and filled it at run time. One example is ScanQRCodeConfirmOpenURLMessage